### ISSUE TITLE
Understanding / replicating learning rate decay

### ISSUE THREAD

**Issue by MrMois** [NVlabs/instant-ngp#1418]
Hello and thanks for publishing this great work!

I am trying to understand how you used the learning rate decay to schedule your training. As the initial learning rate of 1e-2 seems unusual high, a proper decay might be essential to replicate the results. 

In section 5.4 the paper mentions a decay after a given number of steps (20k), but what is a step? Is it:
a) a whole epoch, e.g. iterating over each ray of the training set once
b) a single batch of rays (2^14 rays per batch?)
c) a group of individual sample points evaluated (for example 100 rays with 25 samples each)
d) something else...

I would be very grateful for clarification.

All the best!
